### PR TITLE
Updating Istio Documentation, 

### LINKED ISSUE #1676: [Docs] Update Latest supported version of Istio to 1.7.3
**Current State:**
Currently, the [Meshery Adapter for Istio ](https://meshery.layer5.io/docs/service-meshes/adapters/istio/istio)docs list 1.5 as the latest supported version.

**Desired State:**
v1.7.3 is the latest supported version, though. 

The data file for this adapter needs to be updated.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Maxgit3**
## Istio Documentation Update 1.7.3

**Description**
changed documentation to fit the new version of Istio.

This PR fixes  #1676 

**Notes for Reviewers**
would you like me to look through the Istio documentation to see if anything should be added?

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Maxgit3** [comment]
fixed the sign off issue.  how do you rerun checks?

**Comment by kishore007k** [comment]
> fixed the sign off issue. how do you rerun checks?
Do the following while you are in your branch `Maxgit3:istio-1.7.3-doc-update`
- `git add .`
-  `git commit --amend --signoff`
-  `git push --force-with-lease origin "Your branch name"`

**Comment by Maxgit3** [comment]
the sign off is expecting unknown for name.  not sure why that is.

**Comment by kishore007k** [comment]
> the sign off is expecting unknown for name.  not sure why that is.

Could you reach me on slack?

File: docs/_adapters/istio/istio.md [hunk 1]
```diff
@@ -24,38 +26,39 @@ image: /docs/assets/img/service-meshes/istio.svg
 
 ### Lifecycle management
 
-The {{page.name}} can install **{{page.version}}** of the {{page.name}} service mesh. The SMI adapter for Istio can also be installed using Meshery.
+The Meshery Adapter for Istio can install **v1.7.3** of the Istio service mesh. The SMI adapter for Istio can also be installed using Meshery.
```
Comment (chandrashritii) [inline docs/_adapters/istio/istio.md:29]:
```suggestion
The {{page.name}} can install **{{page.version}}** of the {{page.title}} service mesh. The SMI adapter for Istio can also be installed using Meshery.
```

**Review by chandrashritii — CHANGES_REQUESTED** [review]
Except for the version in the page layout details, the placeholders (for example, {{page.name}}) don't need to be changed.

**Comment by Maxgit3** [comment]
> > the sign off is expecting unknown for name.  not sure why that is.
> 
> Could you reach me on slack?

@kishore007k sorry for the late reply.  yes I can check slack.  what name should I look for?

**Comment by kishore007k** [comment]
> > > the sign off is expecting unknown for name.  not sure why that is.
> > 
> > Could you reach me on slack?
> 
> @kishore007k sorry for the late reply.  yes I can check slack.  what name should I look for?

its kishore

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Pretty 😎, @Maxgit3

**Pull request merged**